code/_onclick/human.dm

```diff
@@ -55,5 +55,9 @@
 		to_chat(src, "<span class='notice'>You try to move your [temp.display_name], but cannot!")
 		return
 
+	if(LAZYACCESS(modifiers, "right"))
+		A.attack_hand_alternate(src)
+		return
+
 	SEND_SIGNAL(src, COMSIG_HUMAN_MELEE_UNARMED_ATTACK, A)
 	A.attack_hand(src)
```

Comment:
add a signal for human_melee_unarmed_attack_right or make it also send that signal already exists, and possibly send the modifiers instead of being a separate unarmed attack

Reply:
Resolved in discord I guess